application/comit_node/src/http_api/swap_resource.rs

```diff
@@ -82,16 +86,19 @@ pub fn build_rfc003_siren_entity<S: StateStore>(
                 .get::<ROLE>(&id)?
                 .ok_or_else(problem::state_store)?;
 
-            // TODO: Implement From<actor::State> for SwapOutcome
```

Comment:
:question: Did you mean to delete this?

Reply:
Yes I deleted it on purpose because I think it is unnecessary and the TODO ticket that is flying around is annoying :see_no_evil: